modules/index/src/main/java/org/apache/ignite/internal/index/IndexBuilder.java

```diff
@@ -85,7 +86,8 @@ class IndexBuilder implements ManuallyCloseable {
      *     <li>Index is built in batches using {@link BuildIndexReplicaRequest}, which are then transformed into {@link BuildIndexCommand}
      *     on the replica, batches are sent sequentially.</li>
      *     <li>It is expected that the index building is triggered by the primary replica.</li>
-     *     <li>If the index has already been built, {@link IndexBuildCompletionListener} will be notified.</li>
+     *     <li>If the index has already been built, {@link IndexBuildCompletionListener#onBuildCompletion} will be notified.</li>
+     *     <li>After index building is complete, {@link IndexBuildCompletionListener#onBuildCompletion} will be notified.</li>
```

Comment:
Maybe rephrase? IMHO now it looks confusing, like a copypaste error (onBuildCompletion in both cases).

Reply:
Okay, tell me which option is better so as not to confuse. It's harder for me to rephrase this because I understand how it works.

Reply:
_If the index has already been built or after the building is complete, onBuildCompletion will be notified_?

Reply:
Nice